TypeCobol.Test/Parser/CodeElements/EXEC.txt

```diff
@@ -3,47 +3,69 @@
 
 [[ExecStatementText]] [3,8:UPDATE]<ExecStatementText> --> [3,8:UPDATE]<ExecStatementText>
 
+[[ExecStatementText]] [9,9: ]<ExecStatementText> --> [9,9: ]<ExecStatementText>
```

Comment:
Whitespace token where removed in last PR #2164 (commit b8da4855039ec51a4bb386fea075554afe254a82), why put them back in this PR ?

Reply:
This file is no longer in the PR, so it's ok.